src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties

```diff
@@ -109,3 +109,7 @@ org.springframework.boot+spring-boot-maven-plugin=2.7.1
 
 org.vafer+jdeb=1.10
 # https://github.com/tcurdt/jdeb/pull/363
+
+org.openapitools+openapi-generator-maven-plugin=-
+# https://github.com/OpenAPITools/openapi-generator/issues/21066
+# workaround: configure the plugin's nested property configOptions.hideGenerationTimestamp to true
```

Comment:
the spirit of this file is to have support for `project.build.outputTimestamp`, so there is no need to check if the user configured the specific property or not

this PR cannot be merged with he current https://github.com/OpenAPITools/openapi-generator/issues/21066 approach